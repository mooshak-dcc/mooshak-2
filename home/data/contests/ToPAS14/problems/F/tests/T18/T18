1000 13 10
AFFFAFAAFF
AVVVVVVVVA
AVVXVVXVTA
AVVVTVVVTA
AVVXVVVVXA
AVVVXVXTTF
AVTVVIIIVF
AVTVIVXVVA
AVVVVVXVVA
AVVVIVVTVA
AVVVVVVVXA
AVVVVVVVVA
AAAAAAAAAA
12
N4
E7
S9
N3
E5
N9
W8
E7
S2
W7
S3
N4