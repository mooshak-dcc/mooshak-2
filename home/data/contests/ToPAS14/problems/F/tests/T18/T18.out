Dentro(5,7):857:D
